src/module/api/update/search/spec.ts

```diff
@@ -0,0 +1,4 @@
+import * as apigents from "@venturemark/apigents";
+
+export type Req = typeof apigents.Update.Search.I; // TODO define this separately without redirecting to the external library
+export type Res = typeof apigents.Update.Search.O; // TODO define this separately without redirecting to the external library
```

Comment:
I have no idea what I am doing here so this needs to sketched out properly I guess.